content/backend/graphql-js/7-subscriptions.md

```diff
@@ -616,4 +616,4 @@ mutation {
 }
 ```
 
-![](https://imgur.com/cYkqy1j)
+![](https://i.imgur.com/hmqRJws.png)
```

Comment:
Hey! Thanks for pointing it out. The issue was caused because the URL was missing an extension. The new screenshot is a little different from the one linked previously, so I am changing it back to the original and adding the extension. 

Once, again thanks for fixing it.

```suggestion
![](https://i.imgur.com/cYkqy1j.png)
```

Reply:
Whoops! I think I'd checked another image as an example to see what was different and must've forgotten to changed the image ID in the URL 🙈